user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_Data_Sources/Get_ad_hoc_data.md

```diff
@@ -14,7 +14,13 @@ An ad hoc data source retrieves external data based on an Automation script that
 
 For more information on how you can create ad hoc data sources, see [Ad hoc data sources](xref:GQI_Ad_hoc_data_sources).
 
-This query data source is available from DataMiner 10.3.0/10.2.4 onwards.
+This query data source is available from DataMiner 10.2.4/10.3.0 onwards.
+
+## Linking the data source to data in the same dashboard or low-code app
+
+From DataMiner 10.2.5/10.3.0 onwards, when you set up an ad hoc data source in a dashboard or low-code app, you can link its arguments to existing data in the same dashboard or low-code app. Depending on the linked data, more information may need to be specified.
+
+For example, if you link to existing query data with a table listing elements, in the *Type* box, you will then need to select whether you want to use a specific data type (e.g. elements) or tables (prior to DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12<!--RN 41075)-->: query rows). Then you will need to select the property you want to use. In most cases, you can select the property in a dropdown list, except if *Type* is set to *Tables*, *Query rows*, or *Script output*, in which case you will have to specify the value yourself. For tables/query rows, when you start typing the value, DataMiner will propose any matching values it can find.
```

Comment:
I wonder if we should actually keep this entire section. It pretty much describes how linking data works in dashboards or apps. To my surprise I didn't find any documentation on this specifically, but linking data is a generic concept used throughout dashboards and apps:

<img width="263" height="284" alt="image" src="https://github.com/user-attachments/assets/2fa3201a-ae4f-4f0d-8934-a301b8889135" />

Reply:
I think it would be better if we keep this for now but add improving the info on how linking data works to our roadmap for Dashboards/LCA documentation improvements, since this will be a bit more work. Do you agree?

Reply:
Agreed,